federation/manifests/federation-apiserver-deployment.yaml

```diff
@@ -23,7 +23,6 @@ spec:
         - --etcd-servers=http://localhost:2379
         - --service-cluster-ip-range={{.FEDERATION_SERVICE_CIDR}}
         - --secure-port=443
-        - --advertise-address={{.FEDERATION_API_HOST}}
```

Comment:
>  I'd say the external address for the federation API server being a static IP address is a special case (local dev and GCE)

Yes. But in that special case it is an IP and it can be advertised that way.

Also, why unnecessarily break the existing behavior?